ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -322,6 +322,13 @@ object Config {
           "Enable preview version of the next Daml-LF language. Should not be used in production."
         )
 
+      opt[Unit]("daml-lf-dev-mode")
```

Comment:
Why not add a single command-line parameter for setting the supported Daml-LF version? For example, the user could switch between the non-stable versions via `--use-daml-lf=early-access` or `--use-daml-lf=dev`. Another option is to have two separate arguments such as `--use-early-access-daml-lf` or `--use-dev-daml-lf`.
What do you think?

Reply:
> Another option is to have two separate arguments such as --use-early-access-daml-lf or --use-dev-daml-lf.

That's how it works right now, but the names are not very nice. We have `--daml-lf-dev-mode-unsafe` (a new, hidden option added in this PR) and `--early-access` (an existing option, added in a different PR today). There were some discussions regarding the naming of the `--early-access` option, I'd leave it to a separate PR to rename both options (cc @mziolekda).

Reply:
@rautenrieth-da I'm fine with renaming the command-line options in a separate PR. Who needs to agree on the new, consistent set of names between sandbox and kvutils-app?